### ISSUE TITLE
Question about track dropout

### ISSUE THREAD

**Issue by violet-sto** [evolutionaryscale/esm#88]
Hi,

Thanks very much for your wonderful work. Could you provide a more detailed description of track dropout introduced in Appendix A.2.2.2? Does "Dropping out" mean that during pre-training, a certain track will be completely masked with a certain probability, or does it restrict the masking to only the tokens within that track according to the defined noise schedule?

Best regards

**Comment by ebetica** [comment]
It's the first, the entire track is masked with a certain probability.
